linera-service/tests/linera_net_tests.rs

```diff
@@ -2449,7 +2362,6 @@ async fn test_open_chain_node_service(config: impl LineraNetConfig) -> Result<()
             service,
             &params,
             &state,
-            &[],
             None,
         )
         .await
```

Comment:
This also sometimes fails thinking `chain2` doesn't exist. I have confirmed this fails on `main` too.